src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected static final int DECIMAL_FORMAT = 100;
+    protected Color color;
```

Comment:
It can be protected but then it will also be accesibble from other classes in the same package, so we need to be mindful of that, and anyways we need getters and seters for that field